Form-Controls/styles.css

```diff
@@ -0,0 +1,79 @@
+body {
+    font-family: Arial, sans-serif;
+    display: flex;
```

Comment:
Do you need all these flex rules? A simple margin:auto on the form would do basically the same thing. 

Try removing as many rules as you can. Do the simplest thing that can possibly work.

Reply:
I have tried as you mentioned but it did not work with auto value. I would appreciate it if there is more information to use auto value. Thank you

Reply:
It's hard to know without seeing the code you tried. Can you edit your code and update the PR with your question?